Makefile

```diff
@@ -2,6 +2,7 @@ IMAGE_NAME=secrets-store-csi-driver-provider-azure
 REGISTRY_NAME ?= upstreamk8sci
 REGISTRY ?= $(REGISTRY_NAME).azurecr.io
 DOCKER_IMAGE ?= $(REGISTRY)/public/k8s/csi/secrets-store/provider-azure
+LOCAL_DOCKER_IMAGE ?= $(IMAGE_NAME)
```

Comment:
Having an additional variable for images can be confusing. I would suggest updating `REGISTRY` to remove the hard coded value would make it more maintainable long term

Reply:
since `LOCAL_DOCKER_IMAGE` is the same as `IMAGE_NAME`, the prior is redundant. I can take out `LOCAL_DOCKER_IMAGE` and just use `IMAGE_NAME`.  I agree that abstracting many of these top level variables to a `.env`  file would allow for more maintainability. Do you believe it's worth to make the changes in this PR or would that belong to a seaprate issue?

Reply:
Also, with the  `?=` syntax it'll only define the hard-coded value if the variable isn't already set. So, if `REGISTRY` is defined in a `.env` file that can be pulled in, then it should take the value that's inside the `.env`

Reply:
Yes removing the `LOCAL_DOCKER_IMAGE` would be good since it is duplicate.  Sorry I missed the `?=` and you are right it can be overwritten.  I don't think there are any required changes for this other than removing `LOCAL_DOCKER_IMAGE`.

Reply:
We should probably look at placing the values for these VARIABLES into the `secrets.env` file that can be used across debugging, and e2e tests. We will need some documentation to guide the contributor as to what should be put into this file since it is explicitly excluded via the `.gitignore` file.